src/features/defi/providers/idle/components/IdleManager/Withdraw/IdleWithdraw.tsx

```diff
@@ -146,7 +149,7 @@ export const IdleWithdraw: React.FC<WithdrawProps> = ({ accountId }) => {
     )
 
   return (
-    <WithdrawContext.Provider value={{ state, dispatch }}>
+    <WithdrawContext.Provider value={value}>
```

Comment:
Technically shallow comparison of props means this was previously safe, but something something not setting precedents